### ISSUE TITLE
R_polar_jupiter constant wrong value

### ISSUE THREAD

**Issue by jorgepiloto** [poliastro/poliastro#837]
🐞 **Problem**

Just found a small error in the `R_polar_jupiter` constant from the `poliastro.constants.general` module. Value should be `6.6854e7` and not  `6.6854`.

The following code should solve the problem:

```python
R_polar_jupiter = Constant(                                                     
    "R_polar_jupiter",                                                          
     "Jupiter polar radius",                                                     
     6.6854e7,                                                                     
     "m",                                                                        
     10000,                                                                      
     "IAU Working Group on Cartographic Coordinates and Rotational Elements: 2009",
     system="si",                                                                
 )
```

**Comment by jorgepiloto** [comment]
Looks a good issue for new contributors since it is easy to solve :smile:

**Comment by priyanshurohilla** [comment]
i want to take up this issue

**Comment by Juanlu001** [comment]
Closed in #838.
